variants/MALYANM200_F103CB/PeripheralPins.c

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
Could be updated to add USB pin mapping:
https://github.com/stm32duino/Arduino_Tools/blob/master/src/genpinmap/Arduino/STM32F103C(8-B)Tx/PeripheralPins.c

Reply:
Also done. (Sorry if this is annoying - it's how I track to make sure I'm addressing feedback.)